Rakefile

```diff
@@ -20,5 +20,9 @@ namespace :db do
     task :verses => :load_config do
       SeedVerses.run
     end
+    desc "Seed devotionals from /db/seeds/devotionals.yml"
```

Comment:
⛏️ Let's have a blank line in between the end of the last task and the start of this one to help readability.